packages/jest-cli/src/cli/args.js

```diff
@@ -66,8 +70,7 @@ const check = argv => {
       'Unrecognized options: ' + unrecognizedOptions.join(', ')
     );
   }
-  return true;
-};
+}
```

Comment:
nit: please add a semicolon.

`npm run lint` will yell at you about this stuff :)